src/views/Home/components/Footer/index.view.tsx

```diff
@@ -1,49 +1,59 @@
 import * as React from "react";
+import { useState } from "react";
+import { ReactComponent as Arrow } from "images/arrow.svg";
+import { footerProps } from "views/Home/props/footer";
 import "./footer.scss";
 
 const FooterComponent: React.FC = () => {
+  const [showScroll, setShowScroll] = useState(false);
+  const checkScrollTop = () => {
+    if (!showScroll && window.pageYOffset > 400) {
+      setShowScroll(true);
+    } else if (showScroll && window.pageYOffset <= 400) {
+      setShowScroll(false);
+    }
+  };
+  window.addEventListener("scroll", checkScrollTop);
+
+  const scrollTop = () => {
+    window.scrollTo({ top: 0, behavior: "smooth" });
+  };
+
```

Comment:
The way we did the footer last year for both the team page and our landing page was keep the scroll effect regardless if you're on mobile or web. We can add line 19 directly to the Arrow SVG as an `onClick` prop. I'm doing the same for the `Team` page

Reply:
This was in anticipation that we'd made that button sticky, but I definitely see the reasoning behind it now! Got you!